src/ControlSystem/Measurements/BNSCenterOfMass.hpp

```diff
@@ -96,26 +103,39 @@ struct BothNSCenters : tt::ConformsTo<protocols::Measurement> {
     using interpolation_target_tag = void;
 
     /// Tags for the arguments to the apply function.
-    using argument_tags =
-        tmpl::list<domain::Tags::Mesh<3>,
-                   Events::Tags::ObserverDetInvJacobian<Frame::ElementLogical,
-                                                        Frame::Inertial>,
-                   grmhd::ValenciaDivClean::TildeD,
-                   domain::Tags::Coordinates<3, Frame::Distorted>>;
+    using argument_tags = tmpl::list<
+        domain::Tags::Mesh<3>, evolution::dg::subcell::Tags::Mesh<3>,
+        Events::Tags::ObserverDetInvJacobian<Frame::ElementLogical,
+                                             Frame::Inertial>,
+        grmhd::ValenciaDivClean::TildeD,
+        domain::Tags::Coordinates<3, Frame::Distorted>,
```

Comment:
just use the Event::Tags::ObserverCoordinates (you are already using the DetInv one...)

Reply:
and Event::Tags::ObserverMesh